### PR TITLE
[#387] Most common currency in summary box

### PR DESCRIPTION
**Pull request by kindly**
Only show the award amount of the most common currency in the summary box.
The rest are relagated to a modal.

### CODE REVIEW

**Comment by Bjwebb** [comment]
Dev instance to test this: http://current.acceptable-license-valid.387-sum-currencies.grantnav-dev.default.threesixtygiving.uk0.bigv.io/

File: grantnav/frontend/templates/search.html [hunk 1]
```diff
@@ -338,6 +356,35 @@ <h1>Search input is not valid</h1>
...
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            <h4 class="modal-title" id="myModalLabel">Award amounts in other currencies</h4>
+          </div>
+          <div class="modal-body">
+            {% for bucket in results.aggregations.currency_stats.buckets %}
+              {% if not forloop.first %}
+                {% with currency=bucket|get:"key"|upper %}
+                  <dl class="dl-horizontal top-stats-search">
+                    <dt>Total {{currency}} grants</dt> <dd>{{bucket|get:"amount_stats"|get:"count"|get_amount}}</dd>
```
Comment (edugomez) [inline grantnav/frontend/templates/search.html]: Given that ```get``` filter defaults to ```None```, I think this chains of ```get```s might _in principle_ give a 500 (_" 'NoneType' object has no attribute 'get' "_)

File: grantnav/frontend/templates/search.html [hunk 2]
```diff
@@ -338,6 +356,35 @@ <h1>Search input is not valid</h1>
...
+                    <dt>Total {{currency}} awarded</dt> <dd>{{currency|currency_symbol}}{{bucket|get:"amount_stats"|get:"sum"|get_amount}}</dd>
+                    <dt>Largest {{currency}} award</dt> <dd>{{currency|currency_symbol}}{{bucket|get:"amount_stats"|get:"max"|get_amount}}</dd>
+                    <dt>Smallest {{currency}} award</dt> <dd>{{currency|currency_symbol}}{{bucket|get:"amount_stats"|get:"min"|get_amount}}</dd>
```
Comment (edugomez) [inline grantnav/frontend/templates/search.html]: Given that ```get``` filter defaults to None, I think these chains of ```get```s might _in principle_ give a 500 (_" 'NoneType' object has no attribute 'get' "_)

**Comment by robredpath** [comment]
@morchickit please can you review this solution and let us know if you're happy for us to merge it in? Ben's link to the dev site above allows you to see it in action!

**Comment by morchickit** [comment]
Ok, couple of questions - 
1. Can link to the non GBP grants somehow? 
2. The other currency that we have now is only USD, correct?

**Comment by morchickit** [comment]
Ignore my last question, according to the stats we also have 

gbp (254307) usd (758) eur (3) chf (1)

**Comment by morchickit** [comment]
Found the CHF grants under 'Human RIghts' and the box looks great! 
screen shot 2018-02-05 at 17 39 32

**Pull request merged**